File: football-ticket-service/src/main/java/com/app/footballticketservice/dto/StatisticTicketDTO.java
package com.app.footballticketservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class StatisticTicketDTO {
    private String matchName;
    private String homeLogo;
    private String awayLogo;
    private Integer totalTicket;
    private Integer totalTicketSold;
    private Integer totalTicketAvailable;
}

File: football-ticket-ui/src/app/share/model/StatisticTicketDTO.ts
export class StatisticTicketDTO {
  constructor(
    public matchName: string = '',
    public homeLogo: string = '',
    public awayLogo: string = '',
    public totalTicket: number = 0,
    public totalTicketSold: number = 0,
    public totalTicketAvailable: number = 0,
  ) {
  }
}
